diff --git a/node_modules/pigeon-maps/lib/index.esm.js b/node_modules/pigeon-maps/lib/index.esm.js
index 6a98674..7c8507f 100644
--- a/node_modules/pigeon-maps/lib/index.esm.js
+++ b/node_modules/pigeon-maps/lib/index.esm.js
@@ -1555,6 +1555,7 @@ _defineProperty(Map, "defaultProps", {
 });
 
 function Marker(props) {
+  // console.log("» Marker", props);
   var width = typeof props.width !== 'undefined' ? props.width : typeof props.height !== 'undefined' ? props.height * 29 / 34 : 29;
   var height = typeof props.height !== 'undefined' ? props.height : typeof props.width !== 'undefined' ? props.width * 34 / 29 : 34;
 
@@ -1636,7 +1637,9 @@ var defaultSvgAttributes = {
   stroke: 'white',
   r: '30'
 };
+
 function PointComponent(props) {
+  // console.log("» PointComponent", props);
   var latLngToPixel = props.latLngToPixel;
   var _props$coordinates = props.coordinates,
       y = _props$coordinates[0],
@@ -1646,11 +1649,23 @@ function PointComponent(props) {
       cx = _latLngToPixel[0],
       cy = _latLngToPixel[1];
 
+  if (props.svgAttributes?.imagePath) {
+    const path = "M"+cx+","+cy+"c"+props.svgAttributes.imagePath.split(/[c|C|L|l|v|V|h|H](.*)/s)[1]
+
+    return /*#__PURE__*/React.createElement("path", {
+      d: path,
+      fill: props.svgAttributes?.fill || "#93c0d099",
+      stroke:  props.svgAttributes?.stroke || "white",
+      strokeWidth: props.svgAttributes?.strokeWidth || "1",
+    });
+  }
+
   return /*#__PURE__*/React.createElement("circle", _objectSpread2({
     cx: cx,
     cy: cy
   }, props.svgAttributes));
 }
+
 function MultiPoint(props) {
   return /*#__PURE__*/React.createElement(React.Fragment, null, props.coordinates.map(function (point, i) {
     return /*#__PURE__*/React.createElement(PointComponent, _objectSpread2(_objectSpread2({}, props), {}, {
@@ -1710,8 +1725,10 @@ function MultiPolygon(props) {
   }));
 }
 function GeometryCollection(props) {
+  // console.log("» GeometryCollection with PointComponent");
   var renderer = {
     Point: PointComponent,
+    // Point: Marker,
     MultiPoint: MultiPoint,
     LineString: LineString,
     MultiLineString: MultiLineString,
@@ -1748,6 +1765,7 @@ function GeometryCollection(props) {
   });
 }
 function GeoJsonFeature(props) {
+  // console.log("☐ GeoJsonFeature esm ", props);
   var _useState = useState(props.hover || false),
       internalHover = _useState[0],
       setInternalHover = _useState[1];
@@ -1788,6 +1806,7 @@ function GeoJsonFeature(props) {
   })));
 }
 function GeoJson(props) {
+  // console.log("☐ GeoJson in index.esm");
   var _props$mapState = props.mapState,
       width = _props$mapState.width,
       height = _props$mapState.height;
diff --git a/node_modules/pigeon-maps/src/overlays/GeoJson.tsx b/node_modules/pigeon-maps/src/overlays/GeoJson.tsx
index a35d315..896d1ee 100644
--- a/node_modules/pigeon-maps/src/overlays/GeoJson.tsx
+++ b/node_modules/pigeon-maps/src/overlays/GeoJson.tsx
@@ -1,5 +1,6 @@
 import React, { CSSProperties, SVGProps, useMemo, useEffect, useState } from 'react'
 import { PigeonProps, Point } from '../types'
+import { Marker } from 'pigeon-maps/src/overlays/Marker'
 
 interface GeoJsonProps extends PigeonProps {
   className?: string
@@ -113,7 +114,8 @@ export function MultiPolygon(props: GeometryProps): JSX.Element {
 
 export function GeometryCollection(props: GeometryProps): JSX.Element {
   const renderer = {
-    Point: PointComponent,
+    // Point: PointComponent,
+    Point: Marker,
     MultiPoint,
     LineString,
     MultiLineString,
@@ -167,6 +169,8 @@ export function GeoJsonFeature(props: GeoJsonProps): JSX.Element {
     payload: props.feature,
   })
 
+  console.log("return GeoJsonFeature", props);
+  
   return (
     <g
       clipRule="evenodd"
@@ -190,6 +194,8 @@ export function GeoJsonFeature(props: GeoJsonProps): JSX.Element {
 export function GeoJson(props: GeoJsonProps): JSX.Element {
   const { width, height } = props.mapState
 
+  console.log("GeoJson in GeoJson");
+  
   return (
     <div
       style={{
